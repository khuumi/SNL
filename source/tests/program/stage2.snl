another_stage:
6.9
done