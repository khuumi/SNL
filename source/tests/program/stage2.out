recipes = [],
stages = [{ sname = "another_stage"
   is_start = false
   locals = []
   body = [Expr (Float 6.9)]}
]